test/config.yaml

```diff
@@ -1,4 +0,0 @@
-sampletable: pasilla_sampletable.tsv
-sample_dir: pasilla
-assembly: dm6
-data_dir: /data/LCDB/references
```

Comment:
ok, should merge PRs in a few.


On Thu, Jun 30, 2016 at 9:27 AM, Ryan Dale <notifications@github.com> wrote:

> In test/config.yaml
> <https://github.com/lcdb/lcdb-workflows/pull/38#discussion_r69131122>:
>
> > @@ -1,4 +0,0 @@
> > -sampletable: pasilla_sampletable.tsv
> > -sample_dir: pasilla
> > -assembly: dm6
> > -data_dir: /data/LCDB/references
>
> I guess leave it for now. I'm planning on building a more general way of
> handling test datasets, so I'll shuffle stuff around during that.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/lcdb/lcdb-workflows/pull/38/files/f9639e14a375f6b78c75a077b54fa74a431aa4a7#r69131122>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABLtcEWkkXx50L4GbLc0hEdHUSKNBPf_ks5qQ8RFgaJpZM4JBIcI>
> .
>